src/schema/v2/articlesByInternalID.ts

```diff
@@ -0,0 +1,21 @@
+import Article from "./article"
+import {
+  GraphQLList,
+  GraphQLString,
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+export const ArticlesByInternalID: GraphQLFieldConfig<void, ResolverContext> = {
```

Comment:
The need here is to fetch a list of articles by a list of IDs.

A better solution is to update positron to accept a list of `ids` as an argument, in which case I would adapt the `articles` type.

If I get time before Wednesday, I'll look into that. For now, this will fetch for each article individually.

Reply:
Fixed! Turns out positron did accept `ids` (I just missed it when looking yesterday 😅). 

I removed this type altogether.